ivadomed/utils.py

```diff
@@ -231,33 +235,60 @@ def structurewise_uncertainty(fname_lst, fname_hard, fname_unc_vox, fname_out):
     data_iou, data_cv, data_avgUnc = np.zeros(data_hard.shape), np.zeros(data_hard.shape), np.zeros(data_hard.shape)
 
     # load all MC simulations and label them
-    data_lst, data_l_lst = [], []
+    data_lst = []
+    data_l_arr = np.array()
```

Comment:
I'm not sure you can do this. `Array()` has mandatory args (see https://numpy.org/doc/stable/reference/generated/numpy.array.html?highlight=array#numpy.array). Has this code run?

Reply:
It doesn't run, I just pushed it to save it during my reflexion, because I was unsure of the way to go (hence the previous discussion)